integration-tests/bats/sql-server.bats

```diff
@@ -1937,3 +1937,15 @@ s.close()
     [ $status -eq 0 ]
     [[ "$output" =~ "0x1C4C4E338AD7442184509D5182816AC3" ]] || false
 }
+
+@test "sql-branch: CALL DOLT_BRANCH -m on session active branch (dolt sql-server)" {
+    cd repo1
+    dolt branch other
+    start_sql_server
+    dolt sql-client -P $PORT -u dolt --use-db '' -q "call dolt_checkout('other'); call dolt_branch('-m', 'main', 'newMain')"
+    run dolt branch
```

Comment:
Does this test pass? you are checking for newOther below.

Also - what it the --use-db with an empty argument supposed to do? This seems odd to me

Reply:
Whoops. Yeah, this isn't right.

Pushed the fixed version.